go/vt/wrangler/schema.go

```diff
@@ -241,6 +241,9 @@ func (wr *Wrangler) CopySchemaShardFromShard(ctx context.Context, tables, exclud
 	if err != nil {
 		return err
 	}
+	if sourceShardInfo.MasterAlias == nil {
+		return fmt.Errorf("The master in source shard doesn't exist")
```

Comment:
Go Readability comment:

The first word should not be capitalized, see: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

The reasoning behind this is that error strings can get chained as they are passed up the stack i.e. the final error message may be the concatenation of several errors e.g. "this is the top most error because: this error happened because: this error happened".

Note that errors are the big exception here. For comments, we strive for proper sentences and punctuation. For example, see this recommendation: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Other notes:
- if your error string does not include a format directive, please use `errors.New()` instead of `fmt.Errorf()`.
- in this case I would actually include a format directive for the keyspace and the shard. This is more user friendly. On top of that, you can also propose a solution. For example: `fmt.Errorf("no master in shard record %v/%v. Consider to run 'vtctl InitShardMaster' in case of a new shard or to reparent the shard to fix the topology data", sourceKeyspace, sourceShard)`